<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainImageWindow</class>
 <widget class="QMainWindow" name="MainImageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="SliceViewPanel" name="panel0" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="SliceViewPanel" name="panel1" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="panel3D" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:red;</string>
      </property>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
     border-image: url(:/root/fltkbutton.png) repeat;
     border-top-width: 9px;
     border-bottom-width: 9px;
     border-left-width: 3px;
     border-right-width: 3px;
     background-origin: content;
     padding-top: -9px;
     padding-bottom: -9px;
     padding-left: -3px;
     padding-right: -3px;
     min-width: 80px;
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
       </property>
       <property name="text">
        <string>Hello World</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/root/Resources/crosshair.gif</normaloff>
         <normalon>:/root/Resources/crosshair.gif</normalon>:/root/Resources/crosshair.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>130</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
     border-image: url(:/root/fltkbutton.png) repeat;
     border-top-width: 8px;
     border-bottom-width: 8px;
     border-left-width: 2px;
     border-right-width: 2px;
     background-origin: content;
     padding-top: -8px;
     padding-bottom: -8px;
     padding-left: -2px;
     padding-right: -2px;
 }

 QToolButton:pressed {
     border-image: url(:/root/fltkbutton_pressed.png) repeat;
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
       </property>
       <property name="text">
        <string>Hello World</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/root/Resources/crosshair3D.gif</normaloff>
         <normalon>:/root/Resources/crosshair.gif</normalon>:/root/Resources/crosshair3D.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="SliceViewPanel" name="panel2" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Images">
     <property name="title">
      <string>Recent Images</string>
     </property>
     <addaction name="actionRecent_1"/>
     <addaction name="actionRecent_2"/>
     <addaction name="actionRecent_3"/>
     <addaction name="actionRecent_4"/>
     <addaction name="actionRecent_5"/>
    </widget>
    <widget class="QMenu" name="menuSave_Image">
     <property name="title">
      <string>Save Image</string>
     </property>
     <addaction name="actionGreyscale_Image"/>
     <addaction name="separator"/>
     <addaction name="actionPreprocessed_Image"/>
     <addaction name="actionLevel_Set_Image"/>
     <addaction name="actionGreyscale_Image_Region"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <widget class="QMenu" name="menuImage_Slice">
      <property name="title">
       <string>Image Slice</string>
      </property>
      <addaction name="actionAxial"/>
      <addaction name="actionCoronal"/>
      <addaction name="actionSagittal"/>
     </widget>
     <addaction name="menuImage_Slice"/>
     <addaction name="actionScreenshot"/>
     <addaction name="actionScreenshot_Series"/>
    </widget>
    <addaction name="actionOpen_Greyscale_Image"/>
    <addaction name="actionOpen_RGB_Image"/>
    <addaction name="menuRecent_Images"/>
    <addaction name="separator"/>
    <addaction name="menuSave_Image"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionUnload_All"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="title">
     <string>Segmentation</string>
    </property>
    <addaction name="actionLoad_from_Image"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuOverlay">
    <property name="title">
     <string>Overlay</string>
    </property>
    <addaction name="actionAdd_Greyscale_Overlay"/>
    <addaction name="actionAdd_RGB_Overlay"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionImage_Contrast"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSegmentation"/>
   <addaction name="menuOverlay"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionOpen_Greyscale_Image">
   <property name="text">
    <string>Open Greyscale Image ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad_from_Image">
   <property name="text">
    <string>Load from Image...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_RGB_Image">
   <property name="text">
    <string>Open RGB Image...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionGreyscale_Image">
   <property name="text">
    <string>Greyscale Image ...</string>
   </property>
  </action>
  <action name="actionPreprocessed_Image">
   <property name="text">
    <string>Preprocessed Image ...</string>
   </property>
  </action>
  <action name="actionLevel_Set_Image">
   <property name="text">
    <string>Level Set Image ...</string>
   </property>
  </action>
  <action name="actionGreyscale_Image_Region">
   <property name="text">
    <string>Greyscale Image Region ...</string>
   </property>
  </action>
  <action name="actionRecent_1">
   <property name="text">
    <string>Recent 1</string>
   </property>
  </action>
  <action name="actionRecent_2">
   <property name="text">
    <string>Recent 2</string>
   </property>
  </action>
  <action name="actionRecent_3">
   <property name="text">
    <string>Recent 3</string>
   </property>
  </action>
  <action name="actionRecent_4">
   <property name="text">
    <string>Recent 4</string>
   </property>
  </action>
  <action name="actionRecent_5">
   <property name="text">
    <string>Recent 5</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionScreenshot_Series">
   <property name="text">
    <string>Screenshot Series</string>
   </property>
  </action>
  <action name="actionAxial">
   <property name="text">
    <string>Axial</string>
   </property>
  </action>
  <action name="actionCoronal">
   <property name="text">
    <string>Coronal</string>
   </property>
  </action>
  <action name="actionSagittal">
   <property name="text">
    <string>Sagittal</string>
   </property>
  </action>
  <action name="actionUnload_All">
   <property name="text">
    <string>Unload All Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionAdd_Greyscale_Overlay">
   <property name="text">
    <string>Add Greyscale Overlay ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="actionAdd_RGB_Overlay">
   <property name="text">
    <string>Add RGB Overlay ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionImage_Contrast">
   <property name="text">
    <string>Image Contrast ...</string>
   </property>
   <property name="toolTip">
    <string>Adjust image contrast</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliceViewPanel</class>
   <extends>QWidget</extends>
   <header>sliceviewpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
